suite-native/device/src/screens/DeviceInfoModalScreen.tsx

```diff
@@ -88,17 +129,66 @@ export const DeviceInfoModalScreen = () => {
                 </ScreenHeaderWrapper>
             }
         >
-            <Card>
-                <HStack spacing="large">
-                    <Image width={92} height={151} source={deviceImage[deviceModel]} />
-                    <Box justifyContent="center">
-                        <Text variant="titleSmall">{deviceName}</Text>
-                        <Text variant="hint">
-                            {translate('deviceInfo.installedFw', { stringifiedFirmwareVersion })}
+            <Box style={applyStyle(contentStyle)}>
+                <Card {...cardAlertProps}>
+                    <HStack spacing="large">
+                        <Image width={92} height={151} source={deviceImage[deviceModel]} />
+                        <Box justifyContent="center">
+                            <Text variant="titleSmall">{deviceName}</Text>
+                            <Text variant="hint">
+                                {translate('deviceInfo.installedFw', {
+                                    version: currentFwVersion,
+                                })}
+                            </Text>
+                        </Box>
+                    </HStack>
+                </Card>
+            </Box>
+            <VStack spacing="medium">
+                <Button
+                    colorScheme="tertiaryElevation0"
+                    onPress={handleAccessoriesClick}
+                    iconRight="arrowUpRight"
+                >
+                    {translate('deviceInfo.goToAccessories')}
+                </Button>
+                {isUpgradable && (
+                    <Button colorScheme="primary" onPress={handleUpdateClick}>
+                        {translate('deviceInfo.updateHowTo.title')}
+                    </Button>
+                )}
+            </VStack>
+            <BottomSheet
```

Comment:
nit: I would put this bottom sheet to separate component file

Reply:
b6eef5a8b